make/default_compiler_options

```diff
@@ -4,7 +4,7 @@
 # - O: Optimization level. Valid values are {0, 1, 2, 3}.
 # - AR: archiver (must specify for cross-compiling)
 ##
-CC = clang++
+CC = g++
```

Comment:
Careful about uploading personal makefile stuff haha (switch this back to clang++).

Using ```git add```/```git commit``` and avoiding ```git commit -a``` can help dodge these issues.

Reply:
Oops, this was of course unintentional. I'll be more careful next time.

Reply:
This needs to be backed out.

Reply:
Wow. I don't even know how I got to this old diff. Please ignore my last comment.